spinnaker-monitoring-third-party/third_party/prometheus/minimal-spinnaker-dashboard.json

```diff
@@ -407,7 +407,7 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "sum($Function(orca:stage:invocations{isComplete=\"false\"}[$SamplePeriod])) by (instance, executionType, cloudProvider)",
+              "expr": "sum($Function(orca:stage:invocations_total{isComplete=\"false\"}[$SamplePeriod])) by (instance, executionType, cloudProvider)",
```

Comment:
This is probably a separate issue, but it doesn't look like the attribute `isComplete` is included in this metric. Causing this panel to never provide any data points

Reply:
There are a few like that. We could remove the `isComplete` label from the query to match the metric, like the below sample: 
```
orca:stage:invocations_total{application="myapp",cloudProvider="kubernetes",endpoint="monitoring",exported_instance="spin-orca-6f9b77fc4d-kcwj5:8083",exported_job="orca",instance="172.30.83.215:8008",job="spin-monitoring",namespace="spinnaker",pod="spin-orca-6f9b77fc4d-kcwj5",service="spin-monitoring",type="deployManifest"}
orca:stage:invocations_total{application="myapp",endpoint="monitoring",exported_instance="spin-orca-6f9b77fc4d-kcwj5:8083",exported_job="orca",instance="172.30.83.215:8008",job="spin-monitoring",namespace="spinnaker",pod="spin-orca-6f9b77fc4d-kcwj5",service="spin-monitoring",type="jenkins"}
```

Reply:
@ewiseblatt What do you think about this approach above?
Not sure if the `isComplete` label has been missed out or removed on purpose at some stage.
Happy to send additional commits and get them in this PR if desired.

Reply:
The dashboard was written for the original orca. The rewrite earlier this year changed some things. Some metrics are the same and some changed, such as this tag no longer being there.

I'll followup offline with you and changes I started to make but had to put on hold before I could bring it to closure. Perhaps you can take it over the finish line.

Reply:
Sure. Thanks very much.